File: SQLCompare.Core/Entities/Compare/CompareResultItemScripts.cs
namespace SQLCompare.Core.Entities.Compare
{
    /// <summary>
    /// Represent the SQL scripts of a specific item
    /// </summary>
    public class CompareResultItemScripts
    {
        /// <summary>
        /// Gets or sets the creation script of the source item
        /// </summary>
        public string SourceCreateScript { get; set; } = string.Empty;

        /// <summary>
        /// Gets or sets the creation script of the target item
        /// </summary>
        public string TargetCreateScript { get; set; } = string.Empty;

        /// <summary>
        /// Gets or sets the alter script
        /// </summary>
        public string AlterScript { get; set; } = string.Empty;
    }
}

File: SQLSchemaCompare.UI/wwwroot/js/Entities/CompareResultItemScripts.ts
/**
 * Represent the SQL scripts of a specific item
 */
// eslint-disable-next-line @typescript-eslint/no-unused-vars
class CompareResultItemScripts {
    /**
     * Gets or sets the creation script of the source item
     */
    public SourceCreateScript?: string;

    /**
     * Gets or sets the creation script of the target item
     */
    public TargetCreateScript?: string;

    /**
     * Gets or sets the alter script
     */
    public AlterScript?: string;
}
